metaflow/plugins/aws/batch/batch_client.py

```diff
@@ -575,6 +582,28 @@ def __init__(self, group, stream, pos=0, sleep_on_no_data=0):
         self._token = None
 
     def _get_events(self):
+        done = RunningJob.is_done
+        status_code = RunningJob.status_code
+        if done:
+            if status_code == 1:
+                return 'Task failed with a general error. \
```

Comment:
Each message is a dict containing at least 'message' and 'timestamp' so this needs to be formatted like that otherwise it won't work when _get_events is called.

Reply:
Would this work for status_code?

```python
status_code = self.info['container'].get('exitCode').values()
```